doc/video.md

```diff
@@ -1,23 +1,25 @@
-# WebRTC Video Conference SDK
+# Genesys Cloud WebRTC SDK Video Conferencing
 
 This SDK supports creating or joining video conferences. Much of the initialization and call controls
 are done using the public API, but are abstracted being api's in this sdk. There is currently no such
 thing as an alerting or incoming video call. All video sessions are treated as a conference to which
 a user must join manually.
```

Comment:
@jensengar I noticed that the documentation doesn't have the new 1-1 support. I did not remember if the sdk did any thing special or if it just handled it like an incoming pending softphone session. I didn't write it up because I am very much spent on writing documentation. But we should get that information into these docs.

Reply:
Good catch. I think we could probably just remove the `There is currently no such
thing as an alerting or incoming video call. All video sessions are treated as a conference to which
a user must join manually.` and call it good for now.